features/admin.workflow-models.v1 /components/create/approval-step.tsx

```diff
@@ -0,0 +1,101 @@
+import { TrashIcon } from "@oxygen-ui/react-icons";
+import React, { FunctionComponent } from "react";
+import List from "@oxygen-ui/react/List";
+import ListItem from "@oxygen-ui/react/ListItem";
+import Grid from "@oxygen-ui/react/Grid";
+import Card from "@oxygen-ui/react/Card";
+import Typography from "@oxygen-ui/react/Typography";
+import { Fab } from "@mui/material";
+import Box from "@oxygen-ui/react/Box";
+import { MultiStepApprovalTemplate } from "./configuration-details-form";
+import { StepUsersList } from "./create-step-users";
+import { StepRolesList } from "./create-step-roles";
+import "./configuration-details-form.scss";
+import { useTranslation } from "react-i18next";
+import { UserBasicInterface } from "@wso2is/admin.core.v1/models/users";
+import { RolesInterface } from "@wso2is/core/models";
+import { ApprovalSteps, ConfigurationsFormValuesInterface } from "../../models/ui";
+
+interface ApprovalStepProps {
```

Comment:
```suggestion
interface ApprovalStepProps extends IdentifiableComponentInterface {
```
These needs to be extended from `IdentifiableComponentInterface` which will give us the `[ "data-componentid" ]` prop which is used in the E2E testing

Reply:
Also add `data-componentid`s for all the elements which is needed for writing E2Es

Reply:
Addressed in 
commit 6b397afd8a31f6b878064b306df9623d5049cceb